tests/baselines/reference/api/typescript.d.ts

```diff
@@ -3354,7 +3354,6 @@ declare namespace ts {
             readFile(fileName: string): string | undefined;
             writeFile(fileName: string, content: string): void;
             fileExists(file: string): boolean;
-            getModuleResolutionCache(): ModuleResolutionCache | undefined;
```

Comment:
Hm, https://github.com/marko-js/language-server/blob/6829c580d16eb1c6128c90a73af36fa4efb64d08/packages/language-server/src/ts-plugin/index.ts#L59 uses it, but that may be the only example I found, so maybe it's fine.

Reply:
They shouldnt have been. Gitlens went in loops but couldnt figure out when and why this was exposed but from looks of it, it should not be exposed at all

Reply:
It was added in https://github.com/microsoft/TypeScript/pull/47388, it seems. @andrewbranch

Reply:
Wait, but it's internal there. Hm, one sec.

Reply:
It’s internal on `LanguageServiceHost`, but not on `Project` 😕

Reply:
Ah, yeah, I just discovered that. I guess it should just be dropped?

Reply:
(To be clear I'm not trying to actually blame anyone, moreso just trying to do the forensics so we know if there was a reason to have it 😅)